File: Numbers/Reverse_Int/ReverseInt.java
public class ReverseInt {
    public static void main(String... args) {
        System.out.println(reverseInt(123) == 321);
        System.out.println(reverseInt(503) == 305);
        System.out.println(reverseInt(500) == 5);
    }

    private static int reverseInt(int x) {
        int n2 = Math.abs(x);
        int r = 0;
        while (n2 > 0) {
            r = r * 10 + n2 % 10;
            n2 /= 10;
        }
        return x < 0 ? -r : r;
    }

}

File: Numbers/Reverse_Int/reverseInt.js
function reverseInt(n) {
    let n2 = Math.abs(n);
    let result = 0;
    while(n2 > 0) {
        result = result * 10 + n2 % 10;
        n2 = Math.floor(n2 / 10);
    }
    return n < 0 ? -result : result;
}

console.log(reverseInt(123) === 321);
console.log(reverseInt(500) === 5);
console.log(reverseInt(503) === 305);